playbooks/init/validate_ca.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Validate CA
+  hosts: oo_first_master
+  any_errors_fatal: true
+  tasks:
+  - name: Validate openshift_master_ca_certificate when defined
+    fail:
+      msg: >
+        If defined, openshift_master_ca_certificate must include two
+        parameters: certfile and keyfile. The certfile parameter must contain
+        only the single certificate that signs the OpenShift Container Platform
+        certificates. If you have intermediate certificates in your chain, you
+        must bundle them into a different file.
+        See https://docs.openshift.org/latest/install_config/redeploying_certificates.html#redeploying-new-custom-ca
+    when:
```

Comment:
Use `with_item` to run this twice for `openshift_master_ca_certificate` and `{{ inventory_hostname }}:/etc/origin/master/ca.crt`?

Reply:
There are subtle differences in the checks as well as the output (recommended solution).